### ISSUE TITLE
Driving Monster Away seems contrived and unnecessary 

### ISSUE THREAD

**Issue by Taker597** [Taker597/WitcherOldWorldComplete#9]
![Screenshot_20230612_132503_MEGA.png](https://github.com/Taker597/WitcherOldWorldComplete/assets/136295246/33390ae6-5d81-461d-a853-d4a63f964a7f)

**Comment by Nadav-N** [comment]
I think the original rule is different actually.
I think the rule states that if you lose your last card as you kill the monster (or witcher) in your fight turn, it's considered the same as if you've won.
The original "driving the monster away rule" is when the monster's attack causes you to lose your last card but the monster is left with 0-1 cards in it's deck.
But I agree with the sentiment of, maybe this shouldn't cause the monster to be driven away, and just consider this situation as if the attacker lost, and in addition to that have some more severe consequences for losing to prevent the attacking player from just retrying in the following turn (since they are probably at a worse position after losing then they were before their initial fight attempt)

**Comment by Taker597** [comment]
You're correct. I misinterpreted. I will edit the post to simply removing the mechanic. Thank you!

**Comment by Nadav-N** [comment]
So one idea (which we both agree on it seems) is simply removing the mechanic.
Would love to hear more thoughts from other people too if anyone has any

**Comment by Taker597** [comment]
> So one idea (which we both agree on it seems) is simply removing the mechanic.
> Would love to hear more thoughts from other people too if anyone has any

Maybe move the monster, but monster suffers a debuff.

Level 1 Debuff - 2 health point debuff
Level 2 Debuff - 1 health point debuff
Level 3 - No debuff

Emphasis the risk and reward while giving an opportunistic mechanism for Witchers.
